src/blog/2024-10-13/index.md

```diff
@@ -0,0 +1,117 @@
+title: Pre-Alpha v9.0 Release Note
+summary: Release note for Neotokyo; Rebuild Pre-Alpha v9.0
+author: nullsystem
+
+# Pre-Alpha v9.0 Release Note
+2024-10-13 nullsystem
+
+* [Download v9.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v9.0-prealpha)
+* [Install NT;RE (client)](/guide/install/)
+* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)
+
+## Known issue
+
+* Player collision has been broken in v9.0-prealpha due to attemping enemy collision, wait for v9.1-prealpha release for the fix
+
+## New features
+
+### VIP and TDM gamemodes
+VIP mode turns CTG capture points into VIP escape points and a random person is selected as a VIP.
+A VIP objective is to escape to the escape points while teammates tries to protect the
+VIP. The enemy instantly wins the round if the VIP gets killed. The VIP have a different player model, stats,
+and is limited to the SMAC (VIP exlusive weapon).
+
+TDM mode turns off both CTG capture points and ghosts, but enables respawning and round time extended to 10.25m.
+
+The new convars for these settings are as follows:
+
+* `neo_vote_game_mode` - 1 by default, Vote on game mode to play. TDM=0, CTG=1, VIP=2
+* `neo_vip_eligible` - 1 by default, Eligible for VIP
+* `sv_neo_vip_ctg_on_death` - 0 by default, Spawn Ghost when VIP dies, continue the game
+* `sv_neo_change_game_type_mid_round` - 1 by default, Allow game type change mid-match
+* `neo_tdm_round_timelimit` - 10.25 by default, TDM round timelimit, in minutes
+* `neo_ctg_round_timelimit` - 3.25 by default, CTG round timelimit, in minutes
+* `neo_vip_round_timelimit` - 3.25 by default, VIP round timelimit, in minutes
+
+### Custom crosshair
+You can now easily customize your crosshair. Inside options there is now a new tab called "Crosshair" and from here
+you can pick between a textured or custom crosshair. All type of crosshair has the ability to change color, while a
```

Comment:
All types of crosshair have the ability to change color

Optional: "All crosshairs' color can be changed, with the custom crosshair having further settings typical of crosshair customization"